spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerProperties.java

```diff
@@ -34,6 +34,8 @@
 	private final Credentials credentials = new Credentials(
 			GcpScope.SPANNER_DATA.getUrl(), GcpScope.SPANNER_ADMIN.getUrl());
 
+	private String host;
```

Comment:
This is a mix of concerns: we are here in a cloud and that fully doesn't look like someone is going to configure a host explicitly in the cloud environment.

I'm pretty OK to have an `emulator-host` property and auto-configure something on the matter, but that's already must be outside of the standard cloud-based auto-configuration.

See what we have with the `GcpPubSubEmulatorConfiguration` as a source for ideas.